src/main/java/com/fvv/bookstore/bean/Book.java

```diff
@@ -1,12 +1,17 @@
 package com.fvv.bookstore.bean;
 
+import java.util.Date;
+
 /**
- * Create on 03/30/2018
+ * Bean class for a Book object
+ * 
+ * Created on 03/20/2018
  * @author Fatima Vasquez
  * @version 1.0
  */
 public class Book {
```

Comment:
Make all bean classes implements `Serializable` since they are used to carry data from database.

read this: https://docs.oracle.com/javase/tutorial/javabeans/advanced/persistence.html

Reply:
Ok!